docs/contrib/keycloak/keycloak.md

```diff
@@ -137,11 +137,11 @@ services:
 ```
 Then, in your REMS config file, you must set something like:
 ```
-:oidc-metadata-url "http://keycloak:8080/auth/realms/my-realm/.well-known/openid-configuration"`.
+:oidc-metadata-url "http://keycloak:8080/auth/realms/rems-idp/.well-known/openid-configuration"`
 ```
 If you instead set it to something like:
 ```
-:oidc-metadata-url "http://localhost:8080/auth/realms/dycons-researcher-idp/.well-known/openid-configuration"
+:oidc-metadata-url "http://localhost:8080/auth/realms/rems-idp/.well-known/openid-configuration"
```

Comment:
Since this is the _wrong_ url do we need to spell it out? Could just say something like "if you use `localhost:8080` instead of `keycloak:8080`, then REMS will ..." Not insisting you fix this, just a thought.